lib/umwelt/client/writer.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Umwelt.Client.Writer do
+  @moduledoc "File writer task"
+
+  use Task
+  require Logger
+
+  alias Umwelt.Client
+
+  def start_link(module) do
+    Task.start_link(__MODULE__, :run, [module])
+  end
+
+  def run(module) do
+    module.code
+    |> Enum.each(fn {path, code} -> write_to_file(path, code) end)
+
+    send(Client.Clone, {:written, module.name})
+  end
+
+  defp write_to_file(path, code) do
+    path = Path.expand(path, "umwelt_raw")
+    path |> Path.dirname() |> File.mkdir_p!()
+
+    case File.read(path) do
+      {:ok, content} when content == code ->
+        Logger.debug("Write #{path}: identical")
+        {path, :identical}
+
+      {:ok, content} ->
+        :ok = File.write!("#{path}_", content)
```

Comment:
```suggestion
        :ok = File.write!(path <> "_", content)
```
Interpolation is technically slower.